test/inductor/test_cpu_repro.py

```diff
@@ -3394,6 +3395,70 @@ def f(a):
         x = torch.rand(4, 5)
         self.common(f, (x,))
 
+    def test_broadcast_scalar_cpp_tile_2d_kernel(self):
```

Comment:
@jgong5 I created this test based on an existing failure from a torchbench model. Ideally the test should be simpler, something like `a[:] = 1`. However its not obvious to me how to force inductor to create a `CppTile2dKernel` and also load a scalar that needs to be broadcasted prior to a vectorised store. Do you have any ideas?

Reply:
Yep, sometimes it is hard to replicate a simpler UT. The way you added the case LGTM.